joe/joe.py

```diff
@@ -24,15 +26,37 @@
 
 
 import os
+import sys
+import platform
+import operator
 
 from docopt import docopt
+from shutil import copyfile
+from os.path import expanduser
+from termcolor import colored
 
 
 __version__ = '0.0.6'
 
 
 _ROOT = os.path.abspath(os.path.dirname(__file__))
-
+PLATFORM = platform.system()
+_HOME = expanduser('~')
+
+
+def get_share_folder():
+    paths = {
+        'Darwin' : os.path.join(_HOME, '.joe'),
+        'Linux' : os.path.join(_HOME, '.joe'),
+        'Windows' : os.path.join(_HOME, '.joe')
+    }
+    path = paths[PLATFORM] or None
```

Comment:
I think what you're looking for here is `path = paths.get(PLATFORM)` to prevent a `KeyError`.

However, I don't think this is necessary. What's wrong with just `path = os.path.join(_HOME, '.joe')`? If I ran this in a Java Virtual Machine, or some machine that may return something not in `paths`, it would be marked as unsupported when that command should always work fine (unless there's some _deep_ problem, in which case python would just raise an exception).